components/up-tls-secret.libsonnet

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Why is this a separate file and not part of the observatorium up component?

Reply:
Done